src/pages/profile/settings/notifications/discord.tsx

```diff
@@ -0,0 +1,20 @@
+import { NextPage } from 'next';
+import React from 'react';
+import UserSettings from '../../../../components/UserProfile/UserSettings';
+import UserNotificationSettings from '../../../../components/UserProfile/UserSettings/UserNotificationSettings';
+import UserNotificationsDiscord from '../../../../components/UserProfile/UserSettings/UserNotificationSettings/UserNotificationsDiscord';
+import useRouteGuard from '../../../../hooks/useRouteGuard';
+import { Permission } from '../../../../hooks/useUser';
+
+const NotificationsPage: NextPage = () => {
+  useRouteGuard(Permission.MANAGE_USERS);
```

Comment:
How do users get to their own management page with this guard?

Reply:
Same goes for the other profile pages

Reply:
Ah whoops, copy-paste error.